packages/eslint-plugin-excel-custom-functions/src/rules/testing.ts

```diff
@@ -0,0 +1,67 @@
+//   /**
```

Comment:
This is a file that's only used to write up testing samples. Testing samples in the test framework are sent in as strings, so it's easier to write it here, copy-paste to the test framework, and comment out this file. It will be removed at the end before the PR is merged.